db/seeds.rb

```diff
@@ -137,7 +140,8 @@
   address: "#{jane.address}",
   status: "Done",
   recipient: jane,
-  helper: cassandra
+  helper: cassandra,
+  completion_date: Faker::Date.between(from: Date.today, to: 1.year.from_now)
```

Comment:
Can you change this to a date in the past? e.g. from 1 month ago, to today.

Same for other 'Done' tasks in the seed. :)